src/apps/VirtualApp/VirtualChat.js

```diff
@@ -72,9 +72,9 @@ class VirtualChat extends Component {
             Janus.debug("Sending message (" + JSON.stringify(body) + ")");
             chatroom.send({"message": body});
           },
-          err: (err) => {
-            console.err("  -- Error attaching plugin...", err);
-            captureException(err, {source: "Textroom", err: `Error attaching textroom plugin: ${err}`});
+          error: (err) => {
+            console.error("  -- Error attaching plugin...", err);
+            captureMessage(`Chatroom error: attach - ${err}`, {source: "Textroom", err}, 'error');
```

Comment:
Like, network failure may cause this so this is message an acceptable code path?